compile.bat

```diff
@@ -0,0 +1 @@
+mvn clean install
```

Comment:
Move these two scripts into **tools/**, this can help us to keep root directory cleaner +
I thinking about creating universal cli for linux and windows